proposals/NNNN-change-registration-manager.md

```diff
@@ -0,0 +1,63 @@
+# change-registration-manager
+
+* Proposal: [SDL-0052](NNNN-change-registration-manager.md)
```

Comment:
You don't assign the proposal a number, this should be `SDL-NNNN` as noted in the template